doc/changelog.rst

```diff
@@ -4,6 +4,8 @@ Changelog
 Version 0.5.6
 -------------
 
+* Change ``warn`` to ``warning`` in `bm_logging.py` .In Python 2, Logger.warn()
```

Comment:
but the logging tutorial mentions for both py2 and py3...warnings.warn() is called to log a deprecation warning.